rules/sample_cpg_pileup.smk

```diff
@@ -0,0 +1,26 @@
+rule cpg_pileup:
+    input:
+        bam = f"samples/{sample}/whatshap/{sample}.{ref}.deepvariant.haplotagged.bam",
+        bai = f"samples/{sample}/whatshap/{sample}.{ref}.deepvariant.haplotagged.bam.bai",
+        reference = config['ref']['fasta'],
+    output:
+        [f"samples/{sample}/cpg_pileup/{sample}.{ref}.{infix}.denovo.{suffix}"
+         for infix in ['combined', 'hap1', 'hap2']
+         for suffix in ['bed', 'bw', 'mincov4.bed', 'mincov4.bw']]
+    log: f"samples/{sample}/logs/cpg_pileup/{sample}.{ref}.log"
+    benchmark: f"samples/{sample}/benchmarks/cpg_pileup/{sample}.{ref}.tsv"
+    params:
+        min_mapq = 1,
+        pileup_mode = "model",
+        model_dir = "workflow/scripts/pb-CpG-tools/pileup_calling_model",
+        modsites = "denovo",
+        prefix = f"samples/{sample}/cpg_pileup/{sample}"
+    threads: 48
+    conda: "envs/pb-cpg-tools.yaml"
+    shell:
+        """
+        (python3 workflow/scripts/pb-CpG-tools/aligned_bam_to_cpg_scores.py \
+            -b {input.bam} -f {input.reference} -o {params.prefix} \
+            -t {threads} -q {params.min_mapq} -m {params.modsites} \
+            -p {params.pileup_mode} -d {params.model_dir}) > {log} 2>&1
```

Comment:
```suggestion
            -p {params.pileup_mode} -d {params.model_dir} -c 10) > {log} 2>&1
```

Reply:
I hesitant to use a minimum coverage of 10. Given that the workflow was designed to work for ~25-30x coverage, sampling bias would put many regions of the genome below this coverage threshold. I'm not yet sure what the appropriate threshold should be.

I do think it would be useful for the pileup script to suppress the `mincov` output if the `-c` value is <=4, just to save space.

Reply:
The change Dan suggests actually outputs the results for cov = 4 (the minimum possible given the model) and 10, which is the standard for a lot of bisulfite analysis that I've seen. So I think the change is reasonable.